extensions/wikia/Email/Controller/ForumController.class.php

```diff
@@ -153,7 +153,10 @@ protected function assertValidReplyParams() {
 	}
 
 	public function getSubject() {
-		return $this->getMessage( $this->getSubjectKey(), $this->titleText )->text();
+		return $this->getMessage( $this->getSubjectKey() )
+			// No need to encode HTML entities in subject
```

Comment:
Wouldn't "We should not encode HTML entities in the subject" be more appropriate in this case?